### DISCUSSION TITLE
Improved permission structure for /time

### DISCUSSION THREAD

**Discussion by LogGits** [Zrips/CMI#4691] [Feature Requests]
## Description of feature/improvement request: ##
Instead of `cmi.command.time.edit` we should be using seperate permission nodes for the different modification types; freeze, unfreeze, set, take, add, realtime, autorealtime, -smooth.

---
### Detailed explanation of what it should do, when and in what situation: ###
Existing permission node `cmi.command.time.edit` should be changed to `cmi.command.time.{modificationType}` whereby a `cmi.command.time.*` would essentially function as `cmi.command.time.edit` currently does.

### Basic explanation as to why this feature is needed and/or useful to have: ###
I give users the ability to set time once a day. The problem here is that with the `edit` permission they can freeze time and do other "Staffer" modifications. A simple permission structure like this would ensure the utmost customisability for the end-user.

**Comment by Zrips** [comment]
Gets implementation of this as of 8.7.12.4 version
